modules/hcp-cli-gateway.adoc

```diff
@@ -32,6 +32,11 @@ $ chmod +x hcp
 $ sudo mv hcp /usr/local/bin/.
 ----
 
+[NOTE]
+====
+If you download the CLI on a Mac computer, you might see a warning about the `hcp` binary file. You need to adjust your security settings to allow the binary file to be run.
+==== 
```

Comment:
Not sure if you want to use this across other modules in the future, but may be worth a snippet if you do. Not really a peer review edit more just a mod docs/reuse suggestion :) feel free to leave it for now.